name-badges/makebadges.py

```diff
@@ -0,0 +1,40 @@
+#!/bin/pyhton3
+
+import sys
+import re
+import lxml.etree as etree
+import subprocess
+
+namesfile = sys.argv[1]
+
+names=open(namesfile).readlines()
+
+def altersvg(file,name,role):
+    svg = open(file, mode="rb").read()
```

Comment:
Depending on how many names you're running on, this might not matter: it would be faster to read the file once and keep the content around than to read it several times. If we can keep the parsed xml tree around too, even better.

Reply:
Another issue with this line is that the file handle is never closed.